builders/txBuilder.go

```diff
@@ -146,11 +130,11 @@ func transactionToNodeTransaction(tx *data.Transaction) (*transaction.Transactio
 	}, nil
 }
 
-func (builder *txBuilder) createUnsignedMessage(arg data.ArgCreateTransaction) ([]byte, error) {
+func (builder *txBuilder) createUnsignedTx(arg data.ArgCreateTransaction) (*data.Transaction, error) {
 	arg.Signature = ""
 	tx := builder.createTransaction(arg)
 
-	return json.Marshal(tx)
+	return tx, nil
```

Comment:
this function never returns a non-nil error. Consider removing the error output parameter.

Reply:
done